images/minio/Makefile

```diff
@@ -0,0 +1,48 @@
+# Copyright 2016 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include ../image.mk
+
+# ====================================================================================
+# Image Build Options
+
+MINIO_IMAGE = $(BUILD_REGISTRY)/minio-$(GOARCH)
+IMAGES = $(MINIO_IMAGE)
```

Comment:
does this `IMAGES` variable get used?

Reply:
it doesn't get used in this file, but it does get used in the included `image.mk`